src/platforms/common/configuration/options.mdx

```diff
@@ -63,16 +63,38 @@ Enabling `debug` mode makes the SDK generate as much diagnostic data as possible
 
 Sets the release. Some SDKs will try to automatically configure a release out of the box but it's a better idea to manually set it to guarantee that the release is in sync with your deploy integrations or source map uploads. Release names are strings, but some formats are detected by Sentry and might be rendered differently. Learn more about how to send release data so Sentry can tell you about regressions between releases and identify the potential source in [the releases documentation](/product/releases/).
 
+<PlatformSection notSupported={["unity"]}>
+
 By default the SDK will try to read this value from the `SENTRY_RELEASE` environment variable (in the browser SDK, this will be read off of the `window.SENTRY_RELEASE` if available).
 
+</PlatformSection>
+
+<PlatformSection supported={["unity"]}>
+
+By default the SDK will read from `Application.productName` and `Application.version` to create the release.
+
+</PlatformSection>
+
 </ConfigKey>
 
 <ConfigKey name="environment">
 
+<PlatformSection notSupported={["unity"]}>
+
 Sets the environment. This string is freeform and not set by default. A release can be associated with more than one environment to separate them in the UI (think `staging` vs `prod` or similar).
 
 By default the SDK will try to read this value from the `SENTRY_ENVIRONMENT` environment variable (except for the browser SDK where this is not applicable).
 
+</PlatformSection>
+
+<PlatformSection supported={["unity"]}>
+
+Sets the environment. This string is freeform and set by default. A release can be associated with more than one environment to separate them in the UI (think `staging` vs `prod` or similar).
+
+By default the SDK will set the environment to `editor` or `production`, depending on `Application.isEditor`.
```

Comment:
What about something like this:

```suggestion
By default, the SDK reports `editor` when running inside the Unity. Otherwise, the default environment is `production`.
```